modules/fbx/doc_classes/EditorSceneFormatImporterUFBX.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
@AThousandShips if you're interested, you can take a look at the documentation. It's very rough.